openmc/mesh.py

```diff
@@ -1,19 +1,21 @@
+import typing
+import warnings
 from abc import ABC, abstractmethod
 from collections.abc import Iterable
 from math import pi
-from numbers import Real, Integral
+from numbers import Integral, Real
 from pathlib import Path
-import typing
-import warnings
-import lxml.etree as ET
+from typing import Iterable, Optional, Sequence, Tuple
```

Comment:
We also import `Iterable` from `collections.abc`; one of them needs to change (maybe better to explicitly type `collections.abc.Iterable` and use `Iterable` to mean `typing.Iterable`)